.github/workflows/main.yaml

```diff
@@ -14,9 +14,10 @@ jobs:
   build:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3.0.2
+
       - name: Set up JDK 11 for running Gradle
-        uses: actions/setup-java@v3
+        uses: actions/setup-java@v3.4.1
```

Comment:
in splunk-otel-java we don't pin setup-java and checkout

Reply:
Yeah, I'll have to update it too